lib/main.js

```diff
@@ -55,15 +56,20 @@ function config (options) {
     if (options.encoding) {
       encoding = options.encoding
     }
+    if (!options.assign) {
```

Comment:
This overrides you default of true if someone does not pass anything.

Reply:
@jcblw I've fixed this in my latest push <https://github.com/motdotla/dotenv/pull/226/commits/c350650b6bbc01ccf4774ccfc8a48beed748b477